src/components/InjectedComponents/PostDialog.tsx

```diff
@@ -16,12 +16,15 @@ import {
     DialogProps,
     Tooltip,
 } from '@material-ui/core'
+import Jimp from 'jimp'
```

Comment:
why not use `ImageData`.

https://mdn.io/ImageData

It is better if you can reduce the extra dependency

Reply:
of course, will change

Reply:
I wasn't aware of it